app/src/main/java/woowacourse/shopping/ui/cart/CartViewModel.kt

```diff
@@ -3,26 +3,33 @@ package woowacourse.shopping.ui.cart
 import androidx.lifecycle.LiveData
 import androidx.lifecycle.MutableLiveData
 import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import io.hyemdooly.di.annotation.InDisk
+import kotlinx.coroutines.launch
 import woowacourse.shopping.data.CartRepository
-import woowacourse.shopping.model.Product
+import woowacourse.shopping.model.CartProduct
 
-class CartViewModel(
-    private val cartRepository: CartRepository,
+class CartViewModel constructor(
+    @InDisk private val cartRepository: CartRepository,
```

Comment:
아래는 사용자 코드의 예시이므로 참고만 하세요!

```suggestion
    @Qualifier("InDiskCartRepository") private val cartRepository: CartRepository,
```